inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/example/UdpClientExample.java

```diff
@@ -92,6 +92,16 @@ public static void main(String[] args) {
         }
     }
 
+    public static String getRandomString(int length) {
+        StringBuffer sb = new StringBuffer();
+        String string = "i am bus test client!";
+        for (int i = 0; i < length; i++) {
+            int number = new Random().nextInt(string.length());
+            sb.append(string.charAt(number));
+        }
+        return sb.toString();
+    }
+
```

Comment:
Class "Random" is not secure. Please use SecureRandom and different seed.

Is getRandomString invalid code?

Reply:
changed to SecureRandom